spec/mongoid/slug_spec.rb

```diff
@@ -471,6 +471,55 @@ module Mongoid
       end
     end
 
+    context 'when block is configured globally' do
+      before do
+        Mongoid::Slug.configure do |c|
+          c.slug = proc{|cur_obj|
```

Comment:
I need a little help here. If we don't use proc how would we get current object? Really I am learning a lot with this pull request. :)